certblob/generate.sh

```diff
@@ -4,16 +4,16 @@ set -euo pipefail
 shopt -s nullglob globstar
 
 # wincrypt.h was downloaded from the following URL:
-# https://raw.githubusercontent.com/reactos/reactos/8ae8083378546fc7d907e489aeebddbe3c8d9399/sdk/include/psdk/wincrypt.h
-# Note that wincrypt.h only has property ID's from Windows XP.
-# certenroll.h appears to have more (from Windows Vista) but it's not in ReactOS.
+# https://github.com/mingw-w64/mingw-w64/blob/master/mingw-w64-headers/include/wincrypt.h
+# wincrypt.h is implemented from  MinGW-w64 and has property ID's upto 127.
```

Comment:
Please clarify that this covers through Windows Vista.  Also there's a double space on this line.

Reply:
Will " Note that wincrypt.h only has property ID's from Windows Vista. " be suffiecient